docs/faq.md

```diff
@@ -3,40 +3,107 @@
 We list some common troubles faced by many users and their corresponding solutions here.
 Feel free to enrich the list if you find any frequent issues and have ways to help others to solve them.
 
+### Installation
```

Comment:
We should re-ordering the questions based on their frequencies and their raised date.

Reply:
yet